src/frontend/src/components/Comment.vue

```diff
@@ -25,8 +25,12 @@
         <v-row>
           <v-col align="left" cols="10" justify="end">
             <div style="float:left;">
-              <v-icon class="mr-1">mdi-hand-heart</v-icon>
-              <span class="subheading mr-2">{{ recommend }}</span>
+              <div class="like-button" v-on:click="toggleLike">
+                <v-icon class="mr-1" v-bind:class="{ liked: likedByMe }"
```

Comment:
사소한거지만 ㅎㅎ v-bind는 생략 가능하대요
```suggestion
                <v-icon class="mr-1" :class="{ liked: likedByMe }"
```

Reply:
맞아요! 취향의 차이긴하지만 ㅎㅎ
다른 코드와 일관성을 위해서 v-bind는 축약형으로 수정했습니다 :)